applications/peripheral_sensor_node/README.rst

```diff
@@ -0,0 +1,337 @@
+The application also stores measurements in a dedicated non-volatile memory partition.
+You can download the stored data over Bluetooth.
+
+Sampling sensors
+================
+
+The application handles the sensor sampling using the Zephyr's :ref:`zephyr:sensor_api`.
+This approach allows to use any sensor available in Zephyr.
+
+By default, you need to use the Peripheral Sensor node Shield for this application.
+
+Sensors shield
+--------------
+
+The Peripheral Sensor node shield is an extension for the nRF54L15 PDK that contains the following sensors:
+
+  * ``ADXL362`` Low power accelerometer
+  * ``BMI270`` Accelerometer and gyroscope
+  * ``BME688`` Environmental sensor
+
+.. figure:: /images/peripheral_sensor_node_shield.png
+   :width: 350px
+   :align: center
+   :alt: Peripheral Sensor node shield
+
+   Peripheral Sensor node shield
+
+This enables you to test the Peripheral Sensor node application on the nRF54L15 PDK.
+
+Pin assignment
+^^^^^^^^^^^^^^
+
+For the exact pin assignment, refer to the following table:
+
++-------------------------+------------+-----------------+
+| nRF54L15 connector pin  | SIGNAL     | Shield function |
++=========================+============+=================+
+| P1.13                   | BMI270 CS  | Chip Select     |
++-------------------------+------------+-----------------+
+| P2.10                   | ADXL362 CS | Chip Select     |
++-------------------------+------------+-----------------+
+| P2.08                   | SPI MOSI   | Serial Data In  |
++-------------------------+------------+-----------------+
+| P2.09                   | SPI MISO   | Serial Data Out |
++-------------------------+------------+-----------------+
+| P2.06                   | SPI SCK    | Serial Clock    |
++-------------------------+------------+-----------------+
+| P1.14                   | GPIO       | SPI Interrupt   |
++-------------------------+------------+-----------------+
+| P1.11                   | TWI SCL    | TWI Clock       |
++-------------------------+------------+-----------------+
+| P1.12                   | TWI SDA    | TWI Data        |
++-------------------------+------------+-----------------+
+
+Shield is an add-on that you can attach to the nRF54L15 PDK to extend its features and functionalities.
+
+If you want to run the application on the nRF54L15 PDK, you need to attach the sensor shield to it.
+
+.. figure:: /images/peripheral_sensor_node_assy.png
+   :width: 350px
+   :align: center
+   :alt: Peripheral Sensor node assembly
+
+   Peripheral Sensor node assembly
+
+The shield must be oriented in a way that allows its silkscreen to read the same way as PDK.
+The shield must not be rotated.
```

Comment:
Couple of questions:

1. Is this shield used only in this application? Can it be used in the future with other nRF54L15 samples or applications?

2. Is it mandatory for running this application? It is weird to see shield discussed before the requirements of the app are even mentioned.
  - If yes, then the shield should be mentioned under Requirements and in Setup: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/templates/application_README.html#setup.
  -  If not, then it should be mentioned under Additional configuration: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/templates/application_README.html#additional-configuration

Reply:
As discussed offline - maybe it would be a good idea to leave this information here for now and when I create a PR for other nrf54l15 docs I could move this part to a separate article under the Working with nrf54L

Reply:
The shield documentation is created in a separate file.
See: https://github.com/nrfconnect/sdk-nrf/pull/16508